thenewboston_node/blockchain/exceptions.py

```diff
@@ -0,0 +1,10 @@
+from django.utils.translation import gettext_lazy as _
+
+from rest_framework import status
+from rest_framework.exceptions import APIException
+
+
+class NotImplemented(APIException):  # noqa: A001
```

Comment:
clashes with built-in exception name. use `APINotImplemented` or `HTTPNotImplemented`